scapy/layers/hsrp.py

```diff
@@ -67,7 +67,7 @@ class HSRPmd5(Packet):
         ByteEnumField("algo", 0, {1: "MD5"}),
         ByteField("padding", 0x00),
         XShortField("flags", 0x00),
-        IPField("sourceip", None),
+        IPField("sourceip", "127.0.0.1"),
```

Comment:
Can you merge #466? That would make the current `None` default value working and more relevant that `"127.0.0.1"`.